source/features/minimize-upload-bar.tsx

```diff
@@ -6,13 +6,13 @@ import features from '../libs/features';
 import * as icons from '../libs/icons';
 
 async function addButton(): Promise<void> {
-	for (const toolbarGroup of select.all('form:not(.rgh-minimize-upload-bar) markdown-toolbar > :last-child')) {
-		toolbarGroup.prepend(
+	for (const toolbarButton of select.all('md-ref')) {
+		toolbarButton.after(
 			<button type="button" className="toolbar-item tooltipped tooltipped-n rgh-upload-btn" aria-label="Attach files">
 				{icons.cloudUpload()}
 			</button>
 		);
-		toolbarGroup.closest('form')!.classList.add('rgh-minimize-upload-bar');
+		toolbarButton.closest('form')!.classList.add('rgh-minimize-upload-bar');
```

Comment:
This class is no longer used. Is it necessary? Will the current code cause duplicates when new comments are loaded?

Reply:
I couldn't get any duplicate buttons. That's why I removed the `form:not(.rgh-minimize-upload-bar)` selector above. 
But this class is still used and necessary in the attached stylesheet.